index.template.html

```diff
@@ -3282,6 +3293,73 @@ <h3>Semantic Annotations</h3>
         of the lamp.
     </p>
 
+</section>
+<section id="semantic-annotations-example-geoloc">
+    <h3>Example III: Geolocation Annotations</h3>
+
+    <p>
+    For some use cases like in building, agraculture, or smart city location based data is required. This
+    information can be provided in the Thing Description in different ways and can be relyed on
+    different kind of location ontologies depending or purpose (e.g., indoor, outdoor).  Sample vocabulary 
+    can be used such as [[WGS84]] or [[schema.org]]. Also see [[sdw-bp]].
+    </p>
+    
+    <p>
+        The TD snippet below uses <code>lat</code> and <code>long</code> from the [[WGS84]] ontology to 
+        provide latitude and longitude metadata at Thing's top level. 
+    </p>
+
+    <pre class="example" id="saref-state-annotation-example">
+{
+    "@context": [
+        "https://www.w3.org/2019/wot/td/v1",
+        {
+            "wgs84": "http://www.w3.org/2003/01/geo/wgs84_pos#"        
```

Comment:
the usual prefix for WGS84 is `geo` (see [prefix.cc](https://prefix.cc/geo))

Reply:
but it might be confusing for readers to use two different vocabularies (WGS84 and schema.org) for the same kind of information (long/lat). WGS84 is very small and entirely included in schema.org. It might be better to use schema.org only.

Reply:
I will change this to geo.

>It might be better to use schema.org only.

Thats right. The intention was to show different examples that uses different ontologies.